regression/cbmc/phi-merge_uninitialized_values/test.c

```diff
@@ -0,0 +1,55 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void dynamicAllocationUninitialized(int trigger)
+{
+  int *obj;
+  obj = (int*)malloc(sizeof(int));
```

Comment:
For symmetry with the Java case this should only be allocated on one branch (i.e. we should allocate within the `if` / `else`)